tests/acceptance/expected-failures-API-on-OCIS-storage.md

```diff
@@ -5,7 +5,14 @@
 -   [apiWebdavProperties1/setFileProperties.feature:32](https://github.com/owncloud/core/blob/master/tests/acceptance/features/apiWebdavProperties1/setFileProperties.feature#L32)
 -   [apiWebdavProperties1/setFileProperties.feature:33](https://github.com/owncloud/core/blob/master/tests/acceptance/features/apiWebdavProperties1/setFileProperties.feature#L33)
 
-### [Checksum feature](https://github.com/owncloud/ocis-reva/issues/196)
+### [Checksum feature](https://github.com/owncloud/ocis/issues/1291)
+-   [apiMain/checksums.feature:24](https://github.com/owncloud/core/blob/master/tests/acceptance/features/apiMain/checksums.feature#L24)
+-   [apiMain/checksums.feature:25](https://github.com/owncloud/core/blob/master/tests/acceptance/features/apiMain/checksums.feature#L25)
+-   [apiMain/checksums.feature:35](https://github.com/owncloud/core/blob/master/tests/acceptance/features/apiMain/checksums.feature#L35)
+-   [apiMain/checksums.feature:36](https://github.com/owncloud/core/blob/master/tests/acceptance/features/apiMain/checksums.feature#L36)
+-   [apiMain/checksums.feature:46](https://github.com/owncloud/core/blob/master/tests/acceptance/features/apiMain/checksums.feature#L46)
+-   [apiMain/checksums.feature:47](https://github.com/owncloud/core/blob/master/tests/acceptance/features/apiMain/checksums.feature#L47)
+-   [apiMain/checksums.feature:50](https://github.com/owncloud/core/blob/master/tests/acceptance/features/apiMain/checksums.feature#L50)
```

Comment:
I am surprised that these need to be added. I don't see where they were also deleted in this file.

Reply:
https://drone.owncloud.com/owncloud/ocis/2900/24/7
```
runsh: Total unexpected passed scenarios throughout the test run:
apiMain/checksums.feature:24
apiMain/checksums.feature:25
apiMain/checksums.feature:35
apiMain/checksums.feature:36
apiMain/checksums.feature:46
apiMain/checksums.feature:47
apiMain/checksums.feature:50
```

@HariBhandari07 those should not be added to expected-failures.